lib/CL/devices/cpu_dbk/dbk_khr_jpeg_cpu.h

```diff
@@ -0,0 +1,95 @@
+/* dbk_khr_jpeg_cpu.h - jpeg defined builtin kernel functions for cpu devices
+
+   Copyright (c) 2024 PoCL developers
+
+   Permission is hereby granted, free of charge, to any person obtaining a copy
+   of this software and associated documentation files (the "Software"), to
+   deal in the Software without restriction, including without limitation the
+   rights to use, copy, modify, merge, publish, distribute, sublicense, and/or
+   sell copies of the Software, and to permit persons to whom the Software is
+   furnished to do so, subject to the following conditions:
+
+   The above copyright notice and this permission notice shall be included in
+   all copies or substantial portions of the Software.
+
+   THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+   IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+   FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+   AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+   LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+   FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
+   IN THE SOFTWARE.
+*/
+
+#ifndef _DBK_KHR_JPEG_CPU_H_
+#define _DBK_KHR_JPEG_CPU_H_
+
+#include "pocl_cl.h"
+
+/**
+ * @brief Do jpeg compression from a rgb image in the first argument and store
+ * the result and size in the second two arguments.
+ * @param program [in] used to retrieve device data
```

Comment:
add an empty line between the param list

Reply:
and should we use \param or @param throughout? I think \param as that's what LLVM prefers.

Reply:
`\param` is used the most in LLVM.

Reply:
I think it should depend on the file type, `/**` is Javadoc style and uses @param. Javadoc style fits pretty well with c style comments. `///` is c++ style so /param fits there. https://www.doxygen.nl/manual/docblocks.html

Reply:
Yep, best use \param throughout since it's LLVM's preference and no point diverging from that for C.